readme.md

```diff
@@ -185,6 +185,17 @@ $ npm install --save-dev branchsite
 
 For new packages, start the `version` field in package.json at `0.0.0` and let `np` bump it to `1.0.0` or `0.1.0` when publishing.
 
+### Release an update to an old major version
+
+To release a minor/patch version for an old major version, create a branch from the major version's git tag and run `np`:
+
+```bash
+$ git checkout -b fix-old-bug v1.0.0 # (1.0.0 being the previous major version)
```

Comment:
Use the long flag of `-b`

Reply:
I didn't see anything about the long flag online, nor in `git checkout --help` (I've also never seen it used).

Reply:
Oh, you're right. Really strange that it doesn't have a long-flag....